Extraction provider readme (#63)
businext/businext.ai: README.md

# businext.ai

> **Bus**iness **in**sight **ext**ractor **a**nd **i**nterpreter

Reviewer: Nooooooooooooo!!!!

Author: With the demo completed, we now know with certainty that we are indeed a Business Insight Extractor and Interpreter. Therefore the `?` must go.

# Configurations

> To set up this project, a Data provider, an Extraction provider and an Interpretation provider must be configured.

Reviewer 2: If we ever add more providers in the future, this list will be easier to update/read

```suggestion
To set up this project, the following must be configured:

- data provider
- extraction provider
- interpretation provider

```

Also it avoids the question of whether to use an Oxford comma

| Environment Variable Name        | Value Format |                                     Example                                     | Explanation                                                                                                                                                                                                     |
| -------------------------------- | :----------: | :-----------------------------------------------------------------------------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `BUSINESS_API_CONFIG`            |     JSON     |    Click [here](README.md#business_api_config-example) to go to the example     | Serialized Configuration object containing your confidential information to access the APIs.                                                                                                                    |

> | `GOOGLE_APPLICATION_CREDENTIALS` |     PATH     | GOOGLE_APPLICATION_CREDENTIALS="/home/user/Downloads/service-account-file.json" | Provides authentication credentials to your application code. For an explanation on how to obtain this file, refer to the Google Vision documentation linked [here](https://cloud.google.com/vision/docs/setup) |

Reviewer 2: Does it work if people put the quotation marks in the env file?

Author: yes
